t/plugin/authz-keycloak.t

```diff
@@ -74,32 +74,7 @@ done
 
 
 
-=== TEST 3: minimal valid configuration with audience
---- config
-    location /t {
-        content_by_lua_block {
-            local plugin = require("apisix.plugins.authz-keycloak")
-            local ok, err = plugin.check_schema({
-                                audience = "foo",
```

Comment:
Better to change it to use client_id instead of removing the test?

Reply:
we have a test case cover `client_id`, ref: https://github.com/apache/apisix/blob/d3d6151c80fcdc162401429bb0920b381d3f2dd7/t/plugin/authz-keycloak.t#L52-L73

so just remove this is ok.